test/cbmc/proofs/MQTTAgent_Init/MQTTAgent_Init_harness.c

```diff
@@ -0,0 +1,58 @@
+/*
+ * FreeRTOS V202011.00
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
```

Comment:
This may not be the right file header. Might also want to change all our file headers to 2021 soon.

Reply:
Yea. Good catch. I think that will be needed for all our repos.